website/docs/quickstart/flink-lake.md

```diff
@@ -23,6 +24,132 @@ We encourage you to use a recent version of Docker and [Compose v2](https://docs
 
 ### Starting required components
 
+<Tabs groupId="lake-tabs">
+  <TabItem value="paimon" label="Paimon" default>
+
+We will use `docker compose` to spin up the required components for this tutorial.
+
+1. Create a working directory for this guide.
+
+```shell
+mkdir fluss-quickstart-flink-paimon
+cd fluss-quickstart-flink-paimon
```

Comment:
Can we just name this directory `fluss-quickstart-paimon`? In the future, we want to introduce Trino and other query engines into this quickstart. So binding to a specific engine is not feasible. The same to the iceberg directory.